src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NumberOfBalls = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
```

Comment:
According to the checklist, you should avoid using static methods and instead create an instance of the class. However, since `Lottery` is not a utility class and you are creating an instance here, this is fine. Just ensure that the `Lottery` class does not contain static methods that should be instance methods.